config/sync/core.entity_view_display.node.stanford_opportunity.teaser.yml

```diff
@@ -1,6 +1,6 @@
 uuid: eacf0e3e-0783-44c6-9012-4c4c9e2b7c47
 langcode: en
-status: false
+status: true
```

Comment:
Does this need to be enabled?

Reply:
We'll have to create a teaser version for the list in D8CORE-7436. I could leave this out and can enable it in that PR instead if that's preferred! 😃

Reply:
Ok just wanted to check